CoreWelcomeView/src/au/gov/asd/tac/constellation/views/welcome/WelcomeViewPane.java

```diff
@@ -198,14 +217,29 @@ public WelcomeViewPane() {
                 }
                 final String text = recentGraphButtons[i].getText();
 
-                final Rectangle2D value = new Rectangle2D(700, 150, 500, 500);
+                final Rectangle2D value = new Rectangle2D(0, 0, 145, 145);
                 final String screenshotFilename = RecentGraphScreenshotUtilities.getScreenshotsDir() + File.separator + text + ".png";
+                final String screenshotFilenameResize = RecentGraphScreenshotUtilities.getScreenshotsDir() + File.separator + text + "Resize" + ".png";
+
                 if (new File(screenshotFilename).exists()) {
-                    final ImageView imageView = new ImageView(new Image("File:/" + screenshotFilename));
+
+                    if (!new File(screenshotFilenameResize).exists()) {
+                        Path source = Paths.get(screenshotFilename);
+                        Path target = Paths.get(screenshotFilenameResize);
+
+                        try (InputStream is = new FileInputStream(source.toFile())) {
+                            resize(is, target, IMAGE_SIZE, IMAGE_SIZE);
+
+                        } catch (Exception ex) {
```

Comment:
exception can be final. Can you also change this to just an `IOException` catch? (that's the only exception `resize` throws